### PR TITLE
m3.txt

### PR DESCRIPTION
**Pull request by rameshshkharat**

### CODE REVIEW

**Comment by anemyogeendar** [comment]
reviewed

**Comment by ghost** [comment]
approved

**Comment by rameshshkharat** [comment]
test1

**Comment by rameshshkharat** [comment]
tesr

**Comment by rameshshkharat** [comment]
test

**Comment by anemyogeendar** [comment]
2nd review

**Review by ghost — APPROVED** [review]
ok

**Review by anemyogeendar — APPROVED** [review]

**Review by anemyogeendar — APPROVED** [review]
approved

**Review by anemyogeendar — APPROVED** [review]
ok

**Comment by anemyogeendar** [comment]
approved

**Review by anemyogeendar — APPROVED** [review]

**Review by anemyogeendar — APPROVED** [review]

File: m3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+magesh creates one more file3
```
Comment (ghost) [inline m3.txt:1]: adsfsa

**Review by ghost — APPROVED** [review]

**Comment by rameshshkharat** [comment]
test

**Pull request merged**